packages/page-statistics/src/App.tsx

```diff
@@ -0,0 +1,82 @@
+import { ApolloClient, InMemoryCache } from '@apollo/client';
+import { randNumber, randRecentDate, randSoonDate } from '@ngneat/falso';
+import { KeygenTable, KeyStatusCard, Logo, ThemeSwitcher } from '@webb-dapp/webb-ui-components';
+import { WebbUIProvider } from '@webb-dapp/webb-ui-components/provider';
+import { FC } from 'react';
+
+const apolloClient = new ApolloClient({
+  cache: new InMemoryCache(),
+  uri: 'http://localhost:4000',
+});
+
+const App: FC = () => {
+  return (
+    <WebbUIProvider>
+      <div className='min-h-[72px] bg-mono-0 dark:bg-mono-180 mb-8 flex justify-between items-center p-4'>
+        <Logo />
+        <ThemeSwitcher />
+      </div>
+
+      <div className='px-8 py-8'>
+        <div className='flex space-x-4'>
+          <KeyStatusCard
+            title='Active Key'
+            titleInfo='Active Key'
```

Comment:
For tooltip content for 'Active Key':

**The public key of the DKG protocol that is currently active.**